content/en/security/cloud_security_management/setup/fargate.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Setting Up Cloud Security Management on AWS Fargate ECS and EKS
+kind: documentation
+private: true
+---
+
+<div class="alert alert-warning">Cloud Security Management on AWS Fargate is in private beta.</div>
+
+Use the following instructions to enable [CSM Threats][1] for Amazon ECS and EKS on AWS Fargate. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][2].
+
+Datadog Cloud Security Management on AWS Fargate includes built-in threat detection for AWS Fargate process and file integrity monitoring (FIM) events as well as [95 out-of-the-box rules][3].
+
+{{< img src="security/csm/csm_fargate_workflow.png" alt="Diagram showing the workflow for Cloud Security Management on AWS Fargate" width="80%">}}
+
+## Prerequisites
+
+* [CSM Enterprise][4] or [CSM Workload Security][5] with the AWS integration configured
+* Access to AWS Management Console
+* AWS Fargate ECS or EKS workloads
+
+### Images
+
+* cws-instrumentation: datadog/cws-instrumentation-dev:cws-instrumentation-beta
+* Datadog-agent: datadog/agent:latest
+
+## Installation
+
+### AWS Console
+
+1. Log in to the [AWS Management Console][6].
+2. Navigate to the ECS section.
+3. On the left menu, click **Task Definitions**, then click **Create new Task Definition with JSON** or choose an existing Fargate task definition.
+4. Use the JSON definition of the AWS CLI method.
```

Comment:
```suggestion
4. Use the JSON definition, or the AWS CLI method.
```
Is this supposed to say "or" the AWS CLI method ? Also would then suggest an anchor link to the AWS CLI method below.

Reply:
Good catch. After checking AWS Console, I'm pretty sure it's the latter.